dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo

```diff
@@ -0,0 +1,115 @@
+    Placement(visible = true, transformation(origin = {-100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Feedback feedback1 annotation(
+    Placement(visible = true, transformation(origin = {0, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain gain1(k = W) annotation(
+    Placement(visible = true, transformation(origin = {-50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain gain2(k = W) annotation(
+    Placement(visible = true, transformation(origin = {50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.Integrator integrator(y_start = DerY0) annotation(
+    Placement(visible = true, transformation(origin = {90, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.Integrator integrator1(y_start = Y0) annotation(
+    Placement(visible = true, transformation(origin = {150, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain gain3(k = 2 * D) annotation(
+    Placement(visible = true, transformation(origin = {50, -40}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+
+  parameter Real DerY0 = 0 "Initial derivative of filter output"
+    annotation (Dialog(group="Initialization"));
+  parameter Real Y0 = 0 "Initial filter output"
+    annotation (Dialog(group="Initialization"));
+
+equation
+  connect(gain.y, feedback.u1) annotation(
+    Line(points = {{-139, 0}, {-108, 0}}, color = {0, 0, 127}));
+  connect(feedback.y, gain1.u) annotation(
+    Line(points = {{-90, 0}, {-62, 0}}, color = {0, 0, 127}));
+  connect(gain1.y, feedback1.u1) annotation(
+    Line(points = {{-38, 0}, {-8, 0}}, color = {0, 0, 127}));
+  connect(feedback1.y, gain2.u) annotation(
+    Line(points = {{10, 0}, {38, 0}}, color = {0, 0, 127}));
+  connect(gain2.y, integrator.u) annotation(
+    Line(points = {{62, 0}, {78, 0}}, color = {0, 0, 127}));
+  connect(integrator.y, integrator1.u) annotation(
+    Line(points = {{102, 0}, {138, 0}}, color = {0, 0, 127}));
+  connect(integrator.y, gain3.u) annotation(
+    Line(points = {{102, 0}, {120, 0}, {120, -40}, {62, -40}}, color = {0, 0, 127}));
+  connect(gain3.y, feedback1.u2) annotation(
+    Line(points = {{39, -40}, {0, -40}, {0, -8}}, color = {0, 0, 127}));
+  connect(integrator1.y, feedback.u2) annotation(
+    Line(points = {{162, 0}, {180, 0}, {180, -80}, {-100, -80}, {-100, -8}}, color = {0, 0, 127}));
+  connect(integrator1.y, y) annotation(
+    Line(points = {{162, 0}, {210, 0}}, color = {0, 0, 127}));
+  connect(u, gain.u) annotation(
+    Line(points = {{-220, 0}, {-162, 0}}, color = {0, 0, 127}));
+
+  annotation(preferredView = "diagram",
+    Documentation(info="<html>
+<p>
+This blocks defines the transfer function between the input u and
+the output y as <em>second order</em> system:
+</p>
+<pre>
+                           k
+   y = ---------------------------------------- * u
```

Comment:
Use the same parameters here as in the model

Reply:
k replaced by K